charts/refinery/values.yaml

```diff
@@ -337,8 +337,8 @@ ingress:
   enabled: false
   labels: {}
   annotations: {}
-    # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
+# ingressClassName: nginx
```

Comment:
Could we add docs here for the deprecated keys as well and mention that they are deprecated?

Reply:
Added some comments indicating that they're deprecated